docs/source/examples/tex.rst

```diff
@@ -1,6 +1,27 @@
 LaTeX
 =================================
 
+The Tex() mobject
++++++++++++++++++++
+Just as you can use `Text()` to add text to your videos, you can use `Tex()` to insert LaTeX.
```

Comment:
```suggestion
Just as you can use ``Text()`` to add text to your videos, you can use ``Tex()`` to insert LaTeX.
```

Reply:
or should that be \`:class:~.Text()\` ?

Reply:
```
:class:`~.Text`
```

Reply:
I went with 
```
``Text()``
```

Reply:
Please use the proper reference to create a link to the `Text` class.